api/services/events/v1/task.proto

```diff
@@ -5,15 +5,15 @@ package containerd.services.events.v1;
 option go_package = "github.com/containerd/containerd/api/services/events/v1;events";
 
 message TaskCreate {
-	string container_id = 1;
+	string id = 1;
```

Comment:
task id and container id are still the same thing in this PR.  Just added exec ids to help abstract out the use of pids for the client